src/components/error-summary/linking-multiple-fields/index.njk

```diff
@@ -0,0 +1,50 @@
+---
+title: Error summary
+layout: layout-example.njk
+---
+
+{% from "date-input/macro.njk" import govukDateInput %}
+{% from "error-summary/macro.njk" import govukErrorSummary %}
+
+{{ govukErrorSummary({
+  "titleText": "There is a problem",
+  "errorList": [
+    {
+      "text": "Enter your date of birth and include a day, month and year",
+      "href": "#dob-year"
```

Comment:
In the guidance it says you need to link to the first field, but we link to the last field, is this intentional?

Reply:
Oh wait, I guess the idea is that the example is of something where you do know the wrong feild....

Reply:
Yep. I've updated the example to make this clearer (only adding the `govuk-input--error` class to the year input)